packages/stems/src/hooks/useMediaQueryListener.ts

```diff
@@ -7,7 +7,7 @@ export const useMediaQueryListener = (mediaQuery: string) => {
   // Updates whether there is a match or not
   // when the media query status changes
   const listener = useCallback(
-    matcher => setIsMatch(matcher.matches),
+    (matcher: any) => setIsMatch(matcher.matches),
```

Comment:
do we have a better type?

Reply:
id love too, but i tried many things and nothing worked. it appears we are using it in a way that's not according to the types...

Reply:
oh, awesome lol